pkg/clients/pubsub.go

```diff
@@ -24,36 +24,64 @@ import (
 
 // PubSubMessenger implements the Messenger interface for Google Cloud pubsub.
 type PubSubMessenger struct {
-	projectID string
-	topicID   string
+	projectID  string
+	topicID    string
+	topicDLQID string
```

Comment:
Should i call this `DLQTopicID`?

Reply:
I would put make this dlqTopicID and prefix the other topicID as well so its clear.

Reply:
bailed on this approach in favor of separate messengers for each. ty!